src/app/lifecycle/lifecycle.component.html

```diff
@@ -83,12 +83,13 @@ <h5>Monitoring &amp; Reporting</h5>
                     <p class="mb-0">Monitoring includes environmental (e.g., <a [routerLink]="['/water-quality']">water quality</a>) testing and assessments of structures (e.g., <a [routerLink]="['/tailings-management']">tailings management facilities</a>). Monitoring activities begin long before a mine is constructed in order to gather baseline information on the site.</p>
                 </div>
             </section>
+            <!-- Hiding this section while there are no related documents to display.
             <section class="card">
                 <h4 class="card-header">Related Documents</h4>
                 <ul class="nv-list">
-                    <li><a href="http://www.amebc.ca/documents/resources-and-publications/publications/archive/BC Mining 101 brochure.pdf" target="_blank">Life Cycle of a Mine (Association for Mineral Exploration BC and Mining Association of BC)</a></li>
```

Comment:
Whats the deal with this section?

Reply:
So the deal is, the link is broken and needs to be removed. However, the PO's said they want to keep the 'Related Documents' section, and hide it if it is empty. At this point, it's empty, so it's commented out as per lines 86 & 92

Reply:
So this is just a placeholder, for now. Do we know if/when related documents will be attached? If it is months or years down the road the usual rule of cleaning out the unused code still applies, I believe...

Reply:
I was specifically told 'hidden', but if you really hate it i can take it out :P